securedrop/crypto_util.py

```diff
@@ -37,6 +37,23 @@
                        'BCDEFGHIJKLMNOPQRSTUVWXYZ')
 
 
+def monkey_patch_delete_handle_status(self, key, value):
+        """Parse a status code from the attached GnuPG process.
+        :raises: :exc:`~exceptions.ValueError` if the status message is unknown.
+        """
+        if key in ("DELETE_PROBLEM", "KEY_CONSIDERED"):
```

Comment:
:+1: `DELETE_PROBLEM` should be the only other status when deleting the key (https://github.com/gpg/gnupg/blob/STABLE-BRANCH-2-2/g10/delkey.c#L80)

Where did you get `KEY_CONSIDERED`?

Reply:
that's from upstream: so this method is https://github.com/isislovecruft/python-gnupg/blob/master/pretty_bad_protocol/_parsers.py#L1015 but with an additional `elif` block for the `PINENTRY_LAUNCHED` case